nilearn/maskers/tests/test_nifti_masker.py

```diff
@@ -219,29 +218,31 @@ def test_4d_single_scan():
 
 
 def test_5d():
-    mask = np.zeros((10, 10, 10))
+    """Test that list of 4D images with last dim=1 are treated as 3d."""
```

Comment:
I think that this docstring is not accurate.

Reply:
Thanks for catching that. It should be fixed now.